Update screenshots to match agreed set

jtreanor opened the pull request:
This updates the screenshots that are taken to the agreed set. This is the equivalent Android PR to https://github.com/wordpress-mobile/WordPress-iOS/pull/9807.

Example output is below.

| File name       | Screenshot  |
| ------------- | ------------ |
| screenshot_1.png | ![screenshot_1_1532530504789](https://user-images.githubusercontent.com/1773641/43209112-8dd4dd66-9034-11e8-8660-0c265033c815.png) |
| screenshot_2.png | ![screenshot_2_1532530494178](https://user-images.githubusercontent.com/1773641/43209113-8dfa29e0-9034-11e8-85c0-d3dcd12df5c5.png) |
| screenshot_3.png | ![screenshot_3_1532530494771](https://user-images.githubusercontent.com/1773641/43209114-8e1f5904-9034-11e8-8c1f-3d7a11c5a1fd.png) |
| screenshot_4.png | ![screenshot_4_1532530512778](https://user-images.githubusercontent.com/1773641/43209116-8e40fffa-9034-11e8-979f-52bc5e4e874a.png) |
| screenshot_5.png | ![screenshot_5_1532530506449](https://user-images.githubusercontent.com/1773641/43209117-8e696ddc-9034-11e8-9d64-89b2f87089ce.png) |


Related issue: https://github.com/wordpress-mobile/WordPress-Android/issues/8006

To test: Screenshot automation (`fastlane screenshots`) still works and now takes the adapted set.

jtreanor commented on WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java:
```diff
@@ -146,6 +146,15 @@ private void navigateReader() {
+                        withId(R.id.toolbar_with_spinner),0)));
+        waitForElementUntilDisplayed(spinner).perform(click());
+
+        ViewInteraction spinnerOption = onView(allOf(withId(R.id.text), withText("Discover")));
```
I don't like hardcoding the string here but since this is dynamically generated, I'm not aware of another way to do this right now. If it becomes a problem as we expand to other locales, I will revisit.

loremattei commented:
Great work James!
I'd align the sorting to the agreed one and show the keyboard on the editor one, in order to make the set the same of the iOS one. What do you think?

jtreanor commented:
@loremattei That makes sense to me. However, for some reason, if fastlane takes a screenshot when the keyboard open, it just captures the app behind it.

I'll look into if there is a way to also capture the keyboard and update here.

jtreanor commented:
@loremattei I've done some research on this and unfortunately my suspicions were correct. It is only possible to take a screenshot of our own app. The keyboard is treated as a separate to our app so it cannot be captured.

loremattei commented:
@jtreanor thanks for checking! I'm still inclined to think that it may be possible.
Did you check/tried other strategies? I was testing Falcon, but Fastlane has support for others that may be better suited for this job.

jtreanor commented:
@loremattei You were right! I have changed the strategy to `UiAutomatorScreenshotStrategy` and it now has the keyboard (screenshots also updated).

Thanks for pointing me in the right direction 😄 

The order is now also updated to match iOS.
